docs/api/expect.md

```diff
@@ -27,6 +27,36 @@ type Awaitable<T> = T | PromiseLike<T>
 `expect` has no effect on testing types, if the expression doesn't have a type error. If you want to use Vitest as [type checker](/guide/testing-types), use [`expectTypeOf`](/api/expect-typeof) or [`assertType`](/api/assert-type).
 :::
 
+## soft
+
+- **Type:** `ExpectStatic & (actual: any) => Assertions`
+
+  `expect.soft` is the same as `expect` but does not terminate test execution when the assertion fails. It will be marked as a test failure. Until the test ends, all errors will be output.
```

Comment:
```suggestion
`expect.soft` functions similarly to `expect`, but instead of terminating the test execution upon a failed assertion, it continues running and marks the failure as a test failure. All errors encountered during the test will be displayed until the test is completed.
```

Reply:
Thank you for the review!